zip-0321.html

```diff
@@ -126,6 +127,8 @@
 zcash:tmEZhbWHTpdKMw5it8YDspUXSMGQyFwovpU?%61mount=1
 zcash:%74mEZhbWHTpdKMw5it8YDspUXSMGQyFwovpU?amount=1</pre>
                     <p>Invalid; percent encoding is only allowed in <code>qchar</code> productions, which do not include addresses, amounts, or parameter names.</p>
+                    <pre>zcash://tmEZhbWHTpdKMw5it8YDspUXSMGQyFwovpU</pre>
+                    <p>Invalid; the grammar does not allow <code>//</code>. ZIP 321 URIs are not "hierarchical URIs" in the sense defined in <a id="id14" class="footnote_reference" href="#rfc3986">3</a> section 1.2.3, and do not have an "authority component".</p>
```

Comment:
Minor nit: in the US, I believe the period would go inside the quotes here. I'm not sure about international rules or ZIP conventions so I just added it as a suggestion.
```suggestion
                    <p>Invalid; the grammar does not allow <code>//</code>. ZIP 321 URIs are not "hierarchical URIs" in the sense defined in <a id="id14" class="footnote_reference" href="#rfc3986">3</a> section 1.2.3, and do not have an "authority component."</p>
```

Reply:
I consistently use "logical quotation", which puts punctuation outside a quotation unless it belongs as part of the quoted text (there's an example in this sentence). See [this Wikipedia article](https://en.wikipedia.org/wiki/Quotation_marks_in_English#Logical_quotation), although I would dispute that it's primarily a British vs American English distinction.

Reply:
I agree that "logical quotation" is correct - quotation marks should not include punctuation that is not part of the quoted value.